.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,20 @@
+Closes # .
```

Comment:
The PR template is where I usually take the greatest offense. People are regularly ignore or simply delete the template. I feel we should make these things clear in the contribution guide rather than have this here and just add noise to the process.

Reply:
Thanks for sharing. While I do disagree here[^1], I'm happy to remove it for simplicity at this early stage. :)

[^1]: It's useful to have a template to remind folks about some core PR requirements, even with clear contribution docs.